examples/semantic_segmentation/README.md

```diff
@@ -29,7 +29,7 @@ $ mpiexec -n <#gpu> python eval_semantic_segmentation_multi.py --dataset citysca
 
 ### ADE20k
 
-| Base model |  Reference | ChainerCV |
+| Base model |  Original | Ours |
```

Comment:
`Base model` -> `Model`
`Dilated ResNet50` -> `PSPNet w/ Dilated ResNet50`

(This is consistent with the table of `Cityscapes` and other tables)